x/btccheckpoint/module.go

```diff
@@ -80,9 +80,8 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
-	if err != nil {
-		panic(err)
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
```

Comment:
Seems we can panic here?

Reply:
thought to keep it consistent with the other ones, this was the only instance that previously panicked. Should we panic on every one of them or just return? Osmosis seems to do the latter, so I followed their approach, but happy to change it if people think the other way is better.

Reply:
Maybe we can panic on every one of them as the errors indicate programming errors